custom_components/versatile_thermostat/translations/fr.json

```diff
@@ -46,6 +46,7 @@
           "heater_entity2_id": "2ème radiateur",
           "heater_entity3_id": "3ème radiateur",
           "heater_entity4_id": "4ème radiateur",
+          "heater_keep_alive": "Intervalle keep-alive du switch en secondes",
```

Comment:
I saw that the word “switch” was already being used elsewhere in the French translation, so I used it here too instead of “interrupteur”, but you tell me! As for “keep-alive”, I wonder if it is a technical term that is more readily understood in English than when translated. Again, you tell me. Google Translate actually suggested some strange things equivalent to _“Change the interval of life maintenance”._ 😖